src/cpp/library/cstdio

```diff
@@ -54,7 +54,7 @@ extern "C" {
 
   int vprintf(const char * format, va_list arg);
 
-  int vsprintf(char * str, const char * format, va_list arg);
+  //int vsprintf(char * str, const char * format, va_list arg); // removed as duplicate of vsprintf in cstdarg
```

Comment:
I suggest to fix it the other way round: remove the definition in cstdarg and use this one here, since this header is where it is [supposed](https://en.cppreference.com/w/cpp/io/c/vfprintf) to be.

Reply:
Done.